libs/execution_context/ebpf_maps.c

```diff
@@ -1011,7 +1222,6 @@ _update_hash_map_entry(
     _In_ ebpf_core_map_t* map, _In_opt_ const uint8_t* key, _In_opt_ const uint8_t* data, ebpf_map_option_t option)
```

Comment:
```suggestion
    _Inout_ ebpf_core_map_t* map, _In_opt_ const uint8_t* key, _In_opt_ const uint8_t* data, ebpf_map_option_t option)
```

Reply:
Filed https://github.com/microsoft/ebpf-for-windows/issues/1746 as there are a lot of cases where we should be using _Inout_ or _In_ const.